code/modules/chemistry/Chemistry-Recipes.dm

```diff
@@ -5492,3 +5492,13 @@
 		mix_phrase = "The mixture emits a sudden whine of static and forms into swirling, many faceted shapes that hurt to look at."
 		result_amount = 2
 		mix_sound = 'sound/effects/radio_sweep1.ogg'
+
+	ehsjuice
+		name = "Juice That Makes Your Head Explode"
+		id = "ehsjuice"
+		result = "ehsjuice"
+		required_reagents = list("blackpowder" = 1, "hard_punch" = 1, "juice_raspberry" = 1, "prions" = 1, "sonicpowder" = 1, "water_holy" = 1)
+		mix_phrase = "The mixture coalesce into a dark red liquid. A strange jingle rings out of it."
+		result_amount = 6
+		mix_sound = "sound/effects/ehsjuicejingle.ogg"
```

Comment:
No, putting random meme bits of youtube videos in the game is not a good idea, even disregarding copyright issues.